data/json/npcs/exodii/exodii_merchant_talk.json

```diff
@@ -638,6 +648,71 @@
       { "text": "Well, I'd better be going.  Bye.", "topic": "TALK_DONE" }
     ]
   },
+  {
+    "id": "TALK_EXODII_MERCHANT_Zomborgs",
+    "type": "talk_topic",
+    "dynamic_line": ".",
+    "//~": "Ah, those things.  They're some of our old robots, must've malfunctioned and such.  Real shame, too; they're not cheap to manufacture.  I'll send someone out to clean them up, we're still working on getting them all.",
+    "responses": [
+      { "text": "Why do they act like zombies when they \"malfunction\"?", "topic": "TALK_EXODII_MERCHANT_Robot_Make" },
+      { "text": "Alright.  Can we talk about something else?", "topic": "TALK_EXODII_MERCHANT_Talk" },
+      { "text": "Well, I'd better be going.  Bye.", "topic": "TALK_DONE" }
+    ]
+  },
+  {
+    "id": "TALK_EXODII_MERCHANT_Robot_Make",
+    "type": "talk_topic",
+    "dynamic_line": ".",
+    "//~": "Well, it has to do with how we make them.  Our technology isn't exactly like yours in the sense of robots.  Instead of a computer chip or something like that driving the whole thing, we use the brains of those infected that keep running around.  Not the ones that died, per say, just the ones that went crazy.  They're too damaged for advanced work, but they take orders very well.  We slap them onto whatever chassis we need at the time, start them up, and set them to work.",
```

Comment:
Because the most likely result of the PC joining the exodii is becoming one of these lobotimized borgs (too much brain damage) the truth of what these robots are (lobotimized humans) is one of the deepest secrets of the exodii and not something for non exodii to know.  At least that is how Erk has stated it in lore&design.

Reply:
I removed references to the use of brains in robots that the player could see, though I left the specialty topic about that within the code for use in the future, when the player could join the Exodii and learn about this. Is that alright?